backend/src/main/java/com/redhat/cloud/notifications/routers/internal/InternalResource.java

```diff
@@ -469,4 +481,29 @@ public Response getDailyDigestTimePreference(@PathParam("orgId") String orgId) {
             return Response.status(NOT_FOUND).build();
         }
     }
+
+    /**
+     * Sends a daily digest command to the engine via a REST request.
+     * @param triggerDailyDigestRequestDto the settings of the digest.
+     */
+    @Consumes(APPLICATION_JSON)
+    @POST
+    @Path("/daily-digest")
+    @RolesAllowed(ConsoleIdentityProvider.RBAC_INTERNAL_USER)
+    public void triggerDailyDigest(@NotNull @Valid final TriggerDailyDigestRequestDto triggerDailyDigestRequestDto) {
+        if (!this.environment.isEnvironmentLocal() && !this.environment.isEnvironmentStage()) {
```

Comment:
This class allows a run in a local env (which makes sense) while `DailyDigestResource`  does not. They should be consistent.

Reply:
You're right! I think I forgot about this one. Done!